src/common/mfu_daos.h

```diff
@@ -37,6 +41,84 @@ typedef struct {
     daos_epoch_t epc;      /* src container epoch */
 } daos_args_t;
 
+#ifdef HDF5_SUPPORT
+/* for oid dataset */
+typedef struct {
+	uint64_t oid_hi;
+	uint64_t oid_low;
+	uint64_t dkey_offset;
+} oid_t;
```

Comment:
@dsikich In the future, we may consider a namespace for these new types/functions to avoid any potential conflicts down the road.